webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9628,6 +9627,35 @@ var userOverride = {
                 return '';
             };
 
+            
+            _this.getMessageCreatedAtTime = function (createdAtTime) {
+                if (TIME_FORMAT_24HOURS) {
+                    
+                    var month = ["Jan", "Feb", "Mar", "April", "May", "June", "July", "Aug", "Sep", "Oct", "Nov", "Dec"];
+                    var msgtime = new Date(createdAtTime);
```

Comment:
has this comment been resolved?
https://github.com/Kommunicate-io/Kommunicate-Web-SDK/pull/833#discussion_r811773038

Reply:
yes